code/game/objects/items/devices/laserpointer.dm

```diff
@@ -5,19 +5,27 @@
 	icon_state = "pointer"
 	inhand_icon_state = "pen"
 	worn_icon_state = "pen"
-	var/pointer_icon_state
 	flags_1 = CONDUCT_1
 	item_flags = NOBLUDGEON
 	slot_flags = ITEM_SLOT_BELT
-	custom_materials = list(/datum/material/iron= SMALL_MATERIAL_AMOUNT * 5, /datum/material/glass= SMALL_MATERIAL_AMOUNT * 5)
+	custom_materials = list(/datum/material/iron = SMALL_MATERIAL_AMOUNT * 5, /datum/material/glass = SMALL_MATERIAL_AMOUNT * 5)
 	w_class = WEIGHT_CLASS_SMALL
-	var/turf/pointer_loc
+	///Currently stored blulespace crystal, if any. Required to use the pointer through walls
+	var/obj/item/stack/ore/bluespace_crystal/crystal_lens
+	///Currently stored micro-laser diode
+	var/obj/item/stock_parts/micro_laser/diode
```

Comment:
what happens if one of these was suddenly Move()d out of the pointer, would the ref remain?